distributed/worker.py

```diff
@@ -773,6 +780,10 @@ async def get_metrics(self):
                 "workers": dict(self.bandwidth_workers),
                 "types": keymap(typename, self.bandwidth_types),
             },
+            active_durations={
+                key: now - self.tasks[key].start_time
+                for key in self.active_threads.values()
+            },
```

Comment:
Maybe we should reclaim the term `executing` from above?  That seems to be derivative state from this